apps/core-app/src/main/modules/box-tool/addon/preview/preview-provider.test.ts

```
@@ -0,0 +1,94 @@
+      durationMs: 1,
+      payload: {
+        abilityId: 'preview.expression.basic',
+        title: '2 + 2',
+        primaryLabel: '结果',
+        primaryValue: '4'
+      }
+    }))
+  }
+}
+
+describe('PreviewProvider', () => {
```

ESLint

[error] `describe`s should begin with lowercase

(test/prefer-lowercase-title)